src/database/migrations/20191001134213-create-user.js

```diff
@@ -64,6 +64,11 @@ module.exports = {
         allowNull: true,
         type: Sequelize.STRING
       },
+      rememberMe: {
+        allowNull: false,
+        defaultValue: false,
+        type: Sequelize.STRING
```

Comment:
- Move this to a new migration file
- The datatype for this column should be `Boolean`